ChangeLog

```diff
@@ -14,6 +24,7 @@
 	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
 	option and example updated
 	
+>>>>>>> eddelbuettel/master
```

Comment:
Not really a plausible story as those files weren't part of master.     

This really does not have to be complicated, and I just explained at work where we migrate some repos from SVN:

- start with a clean slate, ie _no_ repos on your end, local or remote
- fork a remote repository; we consider master to be 'good' and the standard
- you now have a remote copy of master
- clone your remote; you now have a local copy
- maybe run `R CMD build ...; R CMD check ...` to validate tools
- create a branch:  `git checkout -b feature/some-new-stuff`
- do work, run checks, ensure everything works, commit this and push it
- your remote should check this on Travis
- once happy, form a pull request which _will show you again which files are involved_ 
- it also shows you what the diff is
- look it over; if you are really sure it makes then submit it

That should could down on the amount of noise going back and forth.
  

-